pom.xml

```diff
@@ -104,7 +104,7 @@ Go, Swift and Salesforce.com Apex.
 
         <argLine>-Xmx512m -Dfile.encoding=${project.build.sourceEncoding}</argLine>
 
-        <pmd.build-tools.version>3</pmd.build-tools.version>
+        <pmd.build-tools.version>4-SNAPSHOT</pmd.build-tools.version>
```

Comment:
The new build tools ruleset configured CloseResource to use `IOUtils.closeQuietly` as close target...